modules/sor/sorV2/lib/utils/helpers.ts

```diff
@@ -107,6 +107,7 @@ export function getHookState(pool: any): HookState | undefined {
             amp: parseUnits(pool.typeData.amp, 3),
             // 18 decimal precision.
             surgeThresholdPercentage: parseEther(pool.hook.dynamicData.surgeThresholdPercentage),
+            maxSurgeFeePercentage: parseEther(pool.hook.dynamicData.maxSurgeFeePercentage),
```

Comment:
I see this was added at a later stage and was not initially part of the contracts. Was added in maths here: https://github.com/balancer/balancer-maths/commit/eeff3ef8cf1105a0aaa6d96a4c0f8b7a62135256.